File: java/self/Self_problem/armstrong.java
import java.util.*;
public class armstrong {
    public static void main(String[] args) {
        int power =0,arm = 0 ;
        System.out.println("Enter starting number.\n");

        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        System.out.println("Enter last number.\n");
        int b= sc.nextInt();

        for(int i = a;i<=b;i++){

            while (i>=0) {
                power++;
                i/=10;
            }
            while (i>=0) {
                int d = i/10;
                arm += d*power;

            }
        }


    }
}

File: python/Competative/Armstrong.py
a = int(input("Enter a number"))

count  = 0
sum = 0

b = a
while(a>0):
    d = a%10
    count += 1
    a//=10

a = b
while(a>0):
    d = a%10
    sum += d**count
    a//=10
    
if(sum == b):
    print("Armstrong")
else:
    print("Not Armstrong")